eventio/event_io_file.py

```diff
@@ -0,0 +1,65 @@
+import struct
+from .object_header import make_ObjectHeader
+from io import BytesIO
+
+class EventIOObject:
```

Comment:
The member `EventIOObject.headers` is a list of headers, containing the headers of each subitem of this object, correct?

How do I access the header of the object itself? EventIOObject should either have a member header or variables like `version`, `type`, `length` ...